enterprise/websvc.jaxwsmodel/src/org/netbeans/modules/websvc/api/jaxws/wsdlmodel/WsdlPort.java

```diff
@@ -45,7 +47,7 @@ public class WsdlPort implements WSPort {
     }
     
     public List<WsdlOperation> getOperations() {
-        List wsdlOperations = new ArrayList();
+        List<WsdlOperation> wsdlOperations = new ArrayList<>();
```

Comment:
How about rewriting this funtion to

public List<WsdlOperation> getOperations() {
  if(port == null) {
      return emptyList();
  } else {
     return port.getOperations().stream().map(WdslOperation::new).collect(toList());
}